ios/src/TGMapViewController.mm

```diff
@@ -9,11 +9,13 @@
 
 #import "TGMapViewController.h"
 #include "platform_ios.h"
+#include "data/propertyItem.h"
 #include "tangram.h"
 
 @interface TGMapViewController ()
 
-@property (nullable, strong, nonatomic) EAGLContext *context;
+@property (nullable, nonatomic) NSString* scenePath;
```

Comment:
Related to Strings, Apple has generally moved to using `NSURL` for paths on disk. The use of strings is considered a legacy implementation at this point. Any issue with change this to using a NSURL and then updating the delegate to use a URL for the path?

One thing to note is you'll want to use the `fileUrl` versions. Otherwise, NSURL defaults to HTTP-esque semantics and will attempt to percent encode the path as if it was an HTTP URL, which is generally bad for file paths :)